cmd/main.go

```diff
@@ -36,7 +36,7 @@ func main() {
 
 	metrics := prometheus.NewRegistry()
 	metrics.MustRegister(
-		versioncollector.NewCollector("thanos"),
+		versioncollector.NewCollector("prom_scrape_analyzer"),
```

Comment:
Accidentally left this in the change set, it's not strictly related to the HTTP config file support

Reply:
LOL, copy-pasta mistake. Thanks!